realm/realm-library/src/main/java/io/realm/StandardRealmSchema.java

```diff
@@ -110,7 +110,7 @@ public RealmObjectSchema create(String className) {
         if (realm.getSharedRealm().hasTable(internalTableName)) {
```

Comment:
I think we can skip this test if the `createTable()` throws a proper exception against the name conflict.